File: Js/algorithms/selection_sort.js
let massive = [6, 1, 7, 8, 9, 3, 5, 4, 2];
let sortedMassive = [];
length = massive.length;

for (let i = 0; i < length; i++) {
    let smallest = Math.min(...massive)
    sortedMassive.push(massive.splice(massive.indexOf(smallest), 1))
}

console.log(sortedMassive)

File: Python/algorithms/selection_sort.py
massive = [5, 8, 3, 1, 2, 4, 6, 7, 9, 10, 8]
sorted_massive = []
while len(massive):
    length = len(massive)
    minim = min(massive)
    min_index = massive.index(minim)
    massive[min_index], massive[length - 1] = massive[length - 1], massive[min_index]
    sorted_massive.append(massive.pop())

print(sorted_massive)
